go.mod

```diff
@@ -47,6 +47,7 @@ require (
 	github.com/prometheus/client_golang v1.15.0
 	github.com/rs/cors v1.9.0
 	github.com/spf13/cobra v1.7.0
+	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.2
```

Comment:
Not sure why this has changed, it's not a dependency. Try running `go mod tidy` on this branch